client/src/Question.svelte

```diff
@@ -26,18 +26,18 @@
 		} else {
 			dir = "up";
 		}
-		let resp = await fetch(`/api/vote/${question.qid}/${dir}`, {
-			"method": "POST",
-		}).then(r => r.json());
-		votedFor.update(vf => {
+		await fetch(`/api/vote/${question.qid}/${dir}`, {
+			method: "POST"
+		}).then((r) => r.json());
+		votedFor.update((vf) => {
```

Comment:
This actually should probably use the value returned from the API rather than just assuming the toggle went in the direction we expected (in case of concurrent operations). Maybe leave a "TODO" comment?

Reply:
Yea, that makes sense. I'll add a TODO comment for now to keep this PR lint/format specific. But I can look into doing a proper fix in a follow up PR.

Reply:
Added todo in [2966a1e](https://github.com/jonhoo/wewerewondering/pull/11/commits/2966a1e7f7cb33157b72a7d0cb4af813d1eb6cc2)